src/pages/OrganizationPage/getOrgData.ts

```diff
@@ -0,0 +1,46 @@
+import { order_by } from 'lib/gql/__generated__/zeus';
+import { client } from 'lib/gql/client';
+
+export const getOrgData = (orgId: number, address: string) =>
+  client.query(
+    {
+      organizations_by_pk: [
```

Comment:
How do we unify / simplify this with the `src/pages/CirclesPage/getOrgData.ts` version which is basically the same @CryptoGraffe ??

Reply:
just unify em and move into  `src/lib/gql/queries.ts`

Reply:
maybe more appropriate to make a useOrg hook that wraps the react query part too?

Reply:
seems we dont...